pkg/secret-store/builder/vmw_secret_builder.go

```diff
@@ -14,6 +14,9 @@ import (
 	"k8s.io/api/core/v1"
 )
 
+// SecretKind a constant representing the kind of the Secret model
+const SecretKind = "Secret"
+
 // VmwSecretBuilder type
 type VmwSecretBuilder struct {
```

Comment:
not related to this PR, but seems like this name should be changed to dispatchSecretBuilder